src/battle_message.cpp

```diff
@@ -432,36 +440,110 @@ std::string GetItemStartMessage2k(const Game_Battler& source, const lcf::rpg::It
 	return ToString(source.GetName()) + particle + ToString(item.name) + ToString(lcf::Data::terms.use_item);
 }
 
-std::string GetDoubleAttackStartMessage2k3() {
-	return "Double Attack";
+std::string GetDoubleAttackStartMessage2k3(const Game_Battler& source) {
+	if (lcf::Data::terms.easyrpg_battle2k3_double_attack == lcf::Data::terms.kDefaultTerm) {
+		return "Double Attack";
+	}
+	return Utils::ReplacePlaceholders(
+		lcf::Data::terms.easyrpg_battle2k3_double_attack,
+		Utils::MakeArray('S'),
+		Utils::MakeSvArray(source.GetName())
+	);
 }
 
-std::string GetSkillStartMessage2k3(const lcf::rpg::Skill& skill) {
-	return ToString(skill.name);
+std::string GetSkillStartMessage2k3(const Game_Battler& source, const Game_Battler* target, const lcf::rpg::Skill& skill) {
+	StringView target_name = "???";
```

Comment:
As these placeholders are from 2kE: When you use a skill that targets the entire party is the %O also returning ??? there?

Reply:
Tested with RPG_RT 2kE and confirmed that %O returns ??? if a skill targets the entire party.